### DISCUSSION TITLE
Happy Rapid mapping!

### DISCUSSION THREAD

**Discussion by hfu** [UNopenGIS/7#427] [General]
I sometimes use Rapid for quick mapping in spare time.

https://rapideditor.org/edit

Let's share what you are doing with Rapid. 

## My edit history (for convenience; feel free to add yours.)
- [hfu](https://www.openstreetmap.org/user/hfu/history)
- [How did you contribute to OpenStreetMap?](https://hdyc.neis-one.org/)

**Comment by yuiseki** [comment]
I also use Rapid for mapping.
My first use of Rapid was through HOTOSM Tasking Manager.

## What I understood
- there were many situations where Rapid was useful in underpopulated areas
  - it was impressively fast in mapping
- however, Rapid was often completely useless in dense residential areas
  - this is the reason why AIs like fAIr are being developed

## My Editing History
- https://www.openstreetmap.org/user/yuiseki/history

**Comment by hfu** [comment]
Central African Republic の Bangui 付近の Sakai I の buildings を Rapid で complete しました。Sakai II - Sakai IV も同様に complete できそうなので、やってみます。

<img width="679" alt="image" src="https://github.com/UNopenGIS/7/assets/18297/b26c57d2-0c7e-4247-8fa5-8f0346959cff">

この地域は Rapid で綺麗に取れている感じがあり、進めやすい気がしています。

**Comment by hfu** [comment]
I am now the most active OSM mapper in Central African Republic ;-) https://osmstats.neis-one.org/?item=countries&country=Central%20African%20Republic

<img width="942" alt="スクリーンショット 2024-03-24 12 35 04" src="https://github.com/UNopenGIS/7/assets/18297/2ba549aa-6ca9-417d-8d84-c128deac20f8">

**Comment by hfu** [comment]
## List of my favorite areas
- [ ] [エジプト　ギーザ県](https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=19.78/30.27763/30.94723)
- [ ] [サントメ・プリンシペ](https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=19.17/0.25622/6.73812)
- [ ] [中央アフリカ バンギ](https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=19.29/4.34606/18.51934)
- [ ] [ジブチ](https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=13.57/11.5590/43.1301)
- [ ] [アカバ](https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=18.45/29.54642/35.01062)
- [ ] [ヴォルゴグラード](https://rapideditor.org/edit#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=18.97/48.83102/44.61511)
- [ ] [マレーシア クチン](https://rapideditor.org/edit#background=Mapbox&datasets=fbRoads,msBuildings&disable_features=boundaries&map=16.34/1.49916/110.34696)

**Comment by hfu** [comment]
こちらの話、https://github.com/UNopenGIS/qmp/issues/2 で継続しています。
